app/javascript/packs/policies.js

```diff
@@ -1,5 +1,19 @@
-import initPolicies from 'Policies/index'
+// @flow
+
+import { initPolicies as PoliciesWidget } from 'Policies'
 
 document.addEventListener('DOMContentLoaded', () => {
-  window.initPolicies = initPolicies
+  const policiesContainer = document.getElementById('policies')
+
+  if (!policiesContainer) {
```

Comment:
Better something like...
```js
if (!policiesContainer) throw TypeError('Policies Widget needs a valid DOM Element to render')
```

Reply:
Changed it. Anyways, in a future commit I'm wrapping it inside a `ReactWrapper` so it's gonna throw this error.